helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -18,36 +15,28 @@ controller:
     {{- toYaml . | nindent 4 }}
     {{- end }}
 
-  # Required for use with CNI based kubernetes installations (such as ones set up by kubeadm),
-  # since CNI and hostport don't mix yet. Can be deprecated once https://github.com/kubernetes/kubernetes/issues/23920
-  # is merged
-  hostNetwork: true
-
   # Optionally change this to ClusterFirstWithHostNet in case you have 'hostNetwork: true'.
   # By default, while using host network, name resolution uses the host's DNS. If you wish nginx-controller
   # to keep resolving names inside the k8s network, use ClusterFirstWithHostNet.
   dnsPolicy: ClusterFirstWithHostNet
 
```

Comment:
It feels like we should wrap this in `if  .Values.ingressNginx.controller.useHostPort`. Though it seems like it works without it.

Reply:
in this case I think we should do the same for hostNetwork

Reply:
Yes, I agree. Missed that

Reply:
done. Apparently if you have **hostNetwork: true** k8s will also enable and set the hostPort, even if hostPort is disabled in the values file